tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
 envlist =
     py27
-    py36
+    py36-django111
+    py36-django22
 
 [testenv]
 deps =
     py27: Django < 2.0
-    py36: Django
+    py36-django111: Django == 1.11.*
+    py36-django22: Django == 2.2.*
     mock
     py27: pytest < 4.6.5
-    py36: pytest
```

Comment:
Not a big deal, but does this not work without the more specific environments below?

Reply:
Huh, looks like it does